docs/example_projectq.ipynb

```diff
@@ -91,8 +90,7 @@
     }
    ],
    "source": [
-    "engine_list = [ManualMapper(lambda ii: ii)]\n",
-    "engine = MainEngine(backend=projectq_backend, engine_list=engine_list)  # create default compiler (simulator back-end)\n",
+    "engine = MainEngine(backend=projectq_backend)  # create default compiler (simulator back-end)\n",
```

Comment:
Later in this example there is an inconsistency between default output and what is outputted when the example is ran:
> Measured 0,0,0,0,0
> Probabilities: {'00000': 0.484375, '10001': 0.515625}
> version 1.0
>   cQASM generated by Quantum Inspire <class 'quantuminspire.projectq.backend_qx.QIBackend'> class
> qubits 5
> 
> h q[0]
> CNOT q[0], q[4]  

and

> Measured 0,0,0,0,0
> Probabilities: {'00000': 0.529296875, '10001': 0.470703125}
> version 1.0
> cQASM generated by Quantum Inspire <class 'quantuminspire.projectq.backend_qx.QIBackend'> class
> qubits 5
> 
> 
> Rz q[0],3.14159265359
> Rz q[1],3.14159265359
> Rx q[1],1.5707963268
> Rz q[1],1.5707963268
> Rx q[1],10.9955742876
> Rx q[0],1.5707963268
> Rz q[0],1.5707963268
> Rx q[0],10.9955742876
> CZ q[0], q[1]
> Rz q[1],3.14159265359
> Rx q[1],1.5707963268
> Rz q[1],1.5707963268
> Rx q[1],10.9955742876

I know it is not part of this ticket, but since we are commiting the example, it would be nice to fix it. Leave that up to you to decide.

Reply:
In the release proces of the SDK a step is added:
-Run all jupyter notebooks in examples/ to make sure the output is up to date (make sure no errors are shown)
So this is taken care of the next time we release ;)

Reply:
great